src/adapters/suggest.js

```diff
@@ -113,7 +113,13 @@ export default class Suggest {
         if (prefixPoint !== undefined) {
           this.onSelect(prefixPoint);
         } else {
-          const selectedItem = items.find(item => item.id === itemId);
+          const selectedItemIndex = items.findIndex(item => item.id === itemId);
+          let selectedItem = items[selectedItemIndex];
```

Comment:
Another `const` :)
BTW, it may be safer to copy the items props instead of modifying the object on place. No risk of side effects.